### ISSUE TITLE
Questions about your implementation

### ISSUE THREAD

**Issue by cheesama** [wilile26811249/Fastformer-PyTorch#1]
https://github.com/wilile26811249/Fastformer-PyTorch/blob/863bdd140f32149b3c369b54a2a5d160a3ef31d7/Fastformer.py#L29

first of all, thank you for your smart & fast implementation. I'm now studying using this code ^^

Comparing code & architecture image in paper, 

in line 29, you declared p, as repeat_global_query * key.
after that when calculating beta_weight, I think this calculated p value should be used
So, 
beta_weight = torch.softmax(key * self.scale_factor, dim = -1) 
-> 
beta_weight = torch.softmax(p * self.scale_factor, dim = -1)

like this. After that 

global_key = key * beta_weight
->
global_key = p * beta_weight

How about your opinion?

**Comment by wilile26811249** [comment]
Yes, you are right. I found this problem too, but I missed the global key. Thanks.
